src/file/file.service.ts

```diff
@@ -90,9 +90,30 @@ export class FileService {
    * @param fileId - the id of the file.
    * @param partialFile - the partial file.
    */
-  public static updateById(fileId: string, partialFile: Partial<IFile>): Promise<IFile> {
+  public static updateById(fileId: string, partialFile: Partial<IFile>): Promise<boolean> {
     return FilesRepository.updateById(fileId, partialFile);
-  }
+	}
+	
+	/**
+	 * updateMany updates a list of files.
+	 * @param files - List of files to update with their updated fields and their id.
+	 */
+	static async updateMany(files: (Partial<IFile> & { id: string })[]): Promise<{updated: string[], failed: { id: string, error: Error }[]}> {
+		const failedFiles: { id: string, error: Error }[] = [];
+		const updatedFiles: string[] = [];
+		for (let i = 0; i < files.length; i++) {
+			try {
+				const updatedFile = await FilesRepository.updateById(files[i].id, files[i]);
```

Comment:
why not just use updateMany of mongoose?

Reply:
https://docs.mongodb.com/manual/reference/method/db.collection.updateMany/

Reply:
so.. for the logs?